src/main/java/uk/gov/hmcts/divorce/caseworker/event/CaseworkerRequestForInformation.java

```diff
@@ -160,24 +182,49 @@ private boolean isApplicantEmailValid(Applicant applicant) {
     }
 
     private List<String> isEmailValid(CaseData caseData, Applicant applicant) {
-        return isApplicantEmailValid(applicant) ? new ArrayList<>() : getValidationError(caseData, applicant);
+        return isApplicantEmailValid(applicant) ? new ArrayList<>() : getInvalidEmailError(caseData, applicant);
     }
 
     private List<String> areBothEmailsValid(CaseData caseData) {
         List<String> errors = new ArrayList<>();
         boolean app1Valid = isApplicantEmailValid(caseData.getApplicant1());
         boolean app2Valid = isApplicantEmailValid(caseData.getApplicant2());
         if (!app1Valid) {
-            errors.add(getValidationErrorString(caseData, caseData.getApplicant1()));
+            errors.add(getInvalidEmailErrorString(caseData, caseData.getApplicant1()));
         }
         if (!app2Valid) {
-            errors.add(getValidationErrorString(caseData, caseData.getApplicant2()));
+            errors.add(getInvalidEmailErrorString(caseData, caseData.getApplicant2()));
         }
         return errors;
     }
 
-    private List<String> isOtherEmailValid(String email) {
-        return isNotEmpty(email) ? new ArrayList<>() : Collections.singletonList(NO_VALID_EMAIL_ERROR + THIS_PARTY);
+    private List<String> doesEmailMatchApplicantOrSolicitor(CaseData caseData, String email) {
```

Comment:
Does this mean that if I enter some other email that isn't one of the sols or one of the apps that we refuse to do the send? Only I thought the point of having other was that they could send to someone else (not that I know why we do that, and I might have misunderstoor the point of having Other)